core/src/main/java/com/sequenceiq/cloudbreak/service/stack/StackService.java

```diff
@@ -261,18 +259,6 @@ public StackClusterStatusView getStatusByNameOrCrn(NameOrCrn nameOrCrn, Long wor
         return foundStack.orElseThrow(() -> new NotFoundException(String.format(STACK_NOT_FOUND_BY_NAME_OR_CRN_EXCEPTION_MESSAGE, nameOrCrn)));
     }
 
-    @PreAuthorize("hasRole('AUTOSCALE')")
-    public Long getWorkspaceId(String crn) {
-        return stackRepository.findWorkspaceIdByCrn(crn);
-    }
-
-    @PreAuthorize("hasRole('AUTOSCALE')")
-    public Tenant getTenant(String crn) {
-        Workspace workspace = stackRepository.findWorkspaceByCrn(crn).orElseThrow(notFound("workspace", crn));
-        return workspace.getTenant();
-    }
-
-    @PreAuthorize("hasRole('AUTOSCALE')")
```

Comment:
why we removed this?

Reply:
This was again duplicate check, since AutoscaleV4Controller::getAllForAutoscale has     @PreAuthorize("hasRole('AUTOSCALE')").